beater/server.go

```diff
@@ -34,11 +36,18 @@ var (
 	errPOSTRequestOnly = errors.New("only POST requests are supported")
 )
 
+type constructor func() processor.Processor
+
+var Routes = map[string]constructor{
+	"/v1/transactions": transaction.NewProcessor,
+	"/v1/errors":       err.NewProcessor,
+}
+
 func newMuxer(config Config, report reporter) *http.ServeMux {
 	mux := http.NewServeMux()
 
-	for path, p := range processor.Registry.Processors() {
-		handler := appHandler(p, config, report)
+	for path, constructor := range Routes {
```

Comment:
would you mind naming this only `c` or `construc` or similar, to avoid having the same name for the type and the variable?